example.yaml

```diff
@@ -8,7 +8,8 @@ definitions:
 fetcher:
   timeout: 60s                          # deadline to fetch response (will do N retries until deadline passes)
   base-backoff: 10s                     # base backoff period for failures
-  # proxy: user:pass@127.0.0.1:8080     # proxy to talk through
+  # proxies:
```

Comment:
It might be a good idea to call out in a comment that specifying more than one proxy will result in a random proxy selection per-request. That could be confusing if you assumed it was round robin or some other balancing strategy.